script/main.py

```diff
@@ -19,10 +25,160 @@ def init_argparser():
   return args
 
 
-def init_db():  
-  '''Initializing the database'''
+def create_connection(db_file):
+  conn = None
+  try:
+    conn = sqlite3.connect(db_file)
+  except sqlite3.Error as e:
+    print(e)
+  
+  return conn
+
+
+def create_table(conn, create_table_sql):
```

Comment:
It would be better to move SQL text here and call this function create_user_table(conn)